templates/index.pug

```diff
@@ -5,7 +5,7 @@ doctype html
 html(lang="en")
   head
     title JSConf - Conferences for the JavaScript Community
-    link(rel="shortcut icon" href="http://2012.jsconf.us/img/favicon.png")
+    link(rel="shortcut icon" href="https://2012.jsconf.us/img/favicon.png")
```

Comment:
I'm fine with explicitly calling out the https endpoint, but what if we just did this according to the parent protocol?
```
link(rel="shortcut icon" href="//2012.jsconf.us/img/favicon.png")
```

Reply:
https://www.paulirish.com/2010/the-protocol-relative-url/